tests/runtime/regression

```diff
@@ -118,3 +118,10 @@ NAME preserve context pointer
 PROG BEGIN { @test[1] = (uint64)1; } tracepoint:syscalls:sys_enter_kill { if (strcontains(comm, "test")) { @test[(uint64)args.pid] = 1; } if (args.pid == @test[1]) { print((1)); } if (args.pid == @test[1]) { print((1)); exit(); } }
 EXPECT Attaching 2 probes...
 TIMEOUT 1
+
+# In the past, codegen would generate misaligned access for multi-key maps in
+# certain conditions. Make sure that this does not happen. See #3644.
+NAME unaligned multi-key map
+PROG profile:hz:1 { @[pid,kstack]=count(); }
```

Comment:
But this test does not fail on master currently. Do we have a test that does as a result of this change not being in?